main/Repl.hs

```diff
@@ -19,7 +20,10 @@
 {-# OPTIONS_GHC -Wno-unused-matches #-}
 {-# OPTIONS_GHC -Wno-unused-imports #-}
```

Comment:
Aside: Do you happen to know why we disable these warnings?

Reply:
Nice catch, missed that completely - removed and cleaned up leftover imports.